src/server/scripts/Commands/cs_go.cpp

```diff
@@ -148,6 +149,36 @@ class go_commandscript : public CommandScript
         return DoTeleport(handler, { spawnpoint->posX, spawnpoint->posY, spawnpoint->posZ }, spawnpoint->mapid);
     }
 
+    static bool HandleGoCreatureScriptCommand(ChatHandler* handler, std::string_view scriptname)
+    {
+        if (!scriptname.data())
+            return false;
+
+        // Make sure we don't pass double quotes into the SQL query. Otherwise it causes a MySQL error
+        std::string str = scriptname.data(); // Making subtractions to the last character does not with in string_view
+        if (str.front() == '"')
+            str = str.substr(1);
+        if (str.back() == '"')
+            str = str.substr(0, str.size() - 1);
```

Comment:
https://en.cppreference.com/w/cpp/string/basic_string_view/remove_suffix

Reply:
Can you help me change that?